Docm verify options page is displayed after choosing an item in style menu

KRJuuga opened the pull request:
- Create folder WhatMyCarWorthPage in KBBMOOCProject folder in pages folder
- Create file CategoryAndStylePage.java in WhatMyCarWorthPage folder
- Create file MyCarValuePage.java in WhatMyCarWorthPage folder
- Create file OptionPage.java in WhatMyCarWorthPage folder
- Create folder WhatMyCarWorthTest in KBBMOOCProject folder in tests folder
- Create file CategoryAndStylePageTest.java in WhatMyCarWorthTest
- Create file dataJson.java in datadriven folder
- Delete SauceLabsFinalPages folder
- Delete SauceLabsFinalTests folder
- Delete target folder
- Change url in configs.properties to https://staging.kbb.com/whats-my-car-worth/
- Change keyword in file CustomKeyword.java from click to clickThenWait
- Change chromedriver due to different version and system
- Move Assert function to OptionPage 
- Change function wait time to 5s

KRJuuga committed:
Delete SauceLabsFinalTests folder

KRJuuga committed:
Delete SauceLabsFinalPages folder

KRJuuga committed:
- Create folder WhatMyCarWorthPage in KBBMOOCProject folder in pages folder - Create file CategoryAndStylePage.java in WhatMyCarWorthPage folder - Create file MyCarValuePage.java in WhatMyCarWorthPage folder - Create file OptionPage.java in WhatMyCarWorthPage folder - Create folder WhatMyCarWorthTest in KBBMOOCProject folder in tests folder - Create file CategoryAndStylePageTest.java in WhatMyCarWorthTest - Create file dataJson.java in datadriven folder - Delete SauceLabsFinalPages folder - Delete SauceLabsFinalTests folder - Delete target folder - Change url in configs.properties to https://staging.kbb.com/whats-my-car-worth/ - Change keyword in file CustomKeyword.java from click to clickThenWait - Change chromedriver due to different version and system

KRJuuga committed:
Change data in datas.json file

- Create folder WhatMyCarWorthPage in KBBMOOCProject folder in pages folder
- Create file CategoryAndStylePage.java in WhatMyCarWorthPage folder
- Create file MyCarValuePage.java in WhatMyCarWorthPage folder
- Create file OptionPage.java in WhatMyCarWorthPage folder
- Create folder WhatMyCarWorthTest in KBBMOOCProject folder in tests folder
- Create file CategoryAndStylePageTest.java in WhatMyCarWorthTest
- Create file dataJson.java in datadriven folder
- Delete SauceLabsFinalPages folder
- Delete SauceLabsFinalTests folder
- Delete target folder
- Change url in configs.properties to https://staging.kbb.com/whats-my-car-worth/
- Change keyword in file CustomKeyword.java from click to clickThenWait
- Change chromedriver due to different version and system

KRJuuga committed:
Move Assert function to OptionPage - Change function wait time to 5s

namnguyen-itvn commented on demo/src/test/java/com/example/test/core/configs.properties:
```diff
@@ -1,4 +1,4 @@
 browser=CHROME
-url=https://www.saucedemo.com/
+url=https://staging.kbb.com/whats-my-car-worth/
```
url should be effect for all test, all team. should change to specific name

KRJuuga replied:
Renamed url to urlforwhatmycarworth

namnguyen-itvn commented on demo/src/test/java/com/example/test/core/datadriven/dataJson.java:
```diff
@@ -0,0 +1,13 @@
+package com.example.test.core.datadriven;
```
should change the dataJson name to other

KRJuuga replied:
change dataJson name to dataJsonforStyleAndCategoryPage.java

namnguyen-itvn commented on demo/src/test/java/com/example/test/core/helper/ReadConfigPropertyFile.java:
```diff
@@ -5,7 +5,7 @@
 public class ReadConfigPropertyFile {
 
-    public static String propertiesFilePath ="src\\test\\java\\com\\example\\test\\core\\configs.properties";
+    public static String propertiesFilePath ="src//test//java//com//example//test//core//configs.properties";
```
pls change

KRJuuga replied:
Have changed to be suitable for window user

namnguyen-itvn commented on demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java:
```diff
@@ -1,151 +1,94 @@
+        try {
+            waitForElementDisplayed(element);
+            element.click();
+            Thread.sleep(5000);
```
should remove thread.sleep and find the way to handle timeout

KRJuuga replied:
Removed thread sleep and use implicitlyWait

namnguyen-itvn commented on demo/src/test/java/com/example/test/core/resources/datas/datas.json:
```diff
@@ -0,0 +1,9 @@
+[
```
should change the name and write the new datadriven method return data Object

KRJuuga replied:
Rename file datas.json to datasforstyleandcategory.json

namnguyen-itvn commented on demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatMyCarWorthPage/MyCarValuePage.java:
```diff
@@ -0,0 +1,86 @@
+package com.example.test.pages.KBBMOOCProject.WhatMyCarWorthPage;
```
should write the web element locator in separate line then call into a method/function

KRJuuga replied:
Write web element locator in separate line then call into a method/function

namnguyen-itvn commented on demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatMyCarWorthPage/OptionPage.java:
```diff
@@ -0,0 +1,31 @@
+package com.example.test.pages.KBBMOOCProject.WhatMyCarWorthPage;
```
should write the web element locator in separate line then call into a method/function

KRJuuga replied:
Write web element locator in separate line then call into a method/function

namnguyen-itvn commented on demo/src/test/java/com/example/test/tests/KBBMOOCProject/KbbBaseTest.java:
```diff
@@ -11,7 +11,7 @@ public class KbbBaseTest {
     public WebDriver driver;
     public CustomKeyword keyword;
-    public String pathChromeDriver = "src\\test\\java\\com\\example\\test\\core\\drivers\\driver\\chromedriver.exe";
+    public String pathChromeDriver = "src//test//java//com//example//test//resources//driver//chromedriver";
```
it's should revert

KRJuuga replied:
Have changed to be suitable for window user

namnguyen-itvn reviewed (changes requested):
please make your PR have change as the comment inline

KRJuuga committed:
Changes base on comments

Rename file dataJson.java to dataJsonforStyleAndCategoryPage.java
Rename file datas.json to datasforstyleandcategory.json
Rename url to urlforwhatmycarworth
Change path suitable for window user 
Remove thread sleep and use implicitlyWait
Write web element locator in separate line then call into a method/function

KRJuuga committed:
Changed Base on Comments

Rename file dataJson.java to dataJsonforStyleAndCategoryPage.java
Rename file datas.json to datasforstyleandcategory.json
Rename url to urlforwhatmycarworth
Change path suitable for window user 
Remove thread sleep and use implicitlyWait
Write web element locator in separate line then call into a method/function

KRJuuga committed:
Delete dataJson.java

KRJuuga committed:
Delete phpAccount.java

KRJuuga committed:
Delete datas.json

KRJuuga committed:
Delete phpaccountdata.json

KRJuuga committed:
Delete demo/src/test/java/com/example/test/tests/KBBMOOCProject/CarForSaleTests directory

KRJuuga committed:
Delete demo/src/test/java/com/example/test/tests/KBBMOOCProject/WhatMyCarWorthTest directory

KRJuuga committed:
Delete demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatMyCarWorthPage directory

KRJuuga committed:
Fix Conflict to Merge
